gravitee-apim-console-webui/src/management/api/general/subscriptions/list/api-portal-subscription-list.component.spec.ts

```diff
@@ -601,6 +602,16 @@ describe('ApiPortalSubscriptionListComponent', () => {
       expect(await createSubBtn).toBeDefined();
       expect(await createSubBtn.isDisabled()).toEqual(false);
     }));
+
+    it('V1 API should not be able to create subscription', fakeAsync(async () => {
+      await init();
+      await initComponent([], fakeApiV1({ id: API_ID }));
+
+      const harness = await loader.getHarness(ApiPortalSubscriptionListHarness);
+      const createSubBtn = await harness.getCreateSubscriptionButton();
+      expect(await createSubBtn).toBeDefined();
+      expect(await createSubBtn.isDisabled()).toEqual(false);
```

Comment:
Should be true?

Reply:
Good catch ! I have pushed a new fixup commit for you to review